docs/developer-docs/latest/getting-started/quick-start.md

```diff
@@ -1,150 +1,488 @@
+
+### Step 3: Update the `Homepage` single type
+
+It's time to make this blog a bit more yours.
+
+Click on [_Single Types > Homepage_](http://localhost:1337/admin/plugins/content-manager/singleType/application::homepage.homepage) in the left-hand menu. Let's edit this homepage:
+
+1. Hover the picture in the **ShareImage field** and click the **Edit** icon <Fa-PencilAlt />.
+2. Click on **Replace Media** and upload an image from your computer. This image will represent your blog when sharing an article on social medias.
+3. Click **Finish** twice.
+4. At the bottom of the page, update the **Title** to `My Wonderful Strapi Blog` in the **Hero** field group.
+5. Finally, click **Save**.
+
+### Step 4: Restart the servers to reflect latest changes
+
+Gatsby is a static-site generator. It means that you need to restart the servers for changes to appear on the frontend:
+
+1. In your terminal, press `Ctrl-C` to stop the servers.
+2. Make sure you are in the `my-project` folder. If not, type `cd my-project` and press `Enter`.
+3. Restart the servers by typing `yarn develop` (or `npm run develop`) and press `Enter`.
+
+After a few moments, you should see your blog with its updated title running at [http://localhost:8000](http://localhost:8000). The `Hello World!` article you have just created is also visible at the bottom of the page.
+
+![GIF: Updated Gatsby blog frontend](../assets/quick-start-guide/qsg-starters-part2-04-restart_servers.gif)
+
+:::warning CONGRATULATIONS! 🥳
+Now you know how to use Strapi to create and update your blog. Keep on creating amazing content!
+:::
+
+## ⏩  What to do next?
+
+The beauty of using Strapi [starters](https://strapi.io/starters) is that the Strapi backend comes with a frontend ready out-of-the-box. Now you probably want to show your shiny new website to the world! The next step is then to deploy both the Strapi backend and the frontend on the platforms of your choice:
+
+👉 For the Strapi backend, we offer various deployment options — see our [Deployment guides](/developer-docs/latest/setup-deployment-guides/deployment.md).
+
+👉 Deploying the frontend mostly depends on the technology it's based on. The easiest way to deploy your Gatsby blog frontend is probably to [deploy on Gatsby Cloud](https://support.gatsbyjs.com/hc/en-us/articles/360052324714-Connecting-to-Strapi).
+
+:::danger 🤓 To go further with starters
+
+* Read more about the [starters CLI](https://strapi.io/blog/announcing-the-strapi-starter-cli) on our blog.
+* Start another project! We have lots of other [Starters](https://strapi.io/starters) you can use to kickstart your blog, e-commerce, corporate website, or portfolio project.
+
+::::
+
+:::: tab Hands-on
+
+## 🚀  Part 1: Create a new project with Strapi
+
+### Step 1: Run the installation script
```

Comment:
Should we add a short paragraph to mention the CLI installation guide?

Reply:
I'd say this is not the right place, in the Quick Start Guide, but would love to get feedback from @meganelacheny on this as well 🙂